classes/scripts/runner.php

```diff
@@ -608,7 +606,7 @@ public static function enableAutorun($name)
 						{
 							$autorunner->writeError($exception->getMessage());
 
-							exit($exception->getCode());
+							exit(2);
```

Comment:
Fair point :)

I don't really know if using different statuses for error and exception is valuable... As you said, the only relevant information is atoum failed dur to an unhandled error (be it an error or an exception). The only useful thing would be to differentiate those failure from regular (handled) failures (those coming from assertions).